terra_notebook_utils/table.py

```diff
@@ -39,6 +39,26 @@ def fetch_drs_url(table: str, file_name: str):
         raise ValueError(f"Expected DRS url in {table} for {file_name}, got {val} instead.")
     return val
 
+def list_entities(ent_type):
+    resp = fiss.fapi.get_entities_with_type(WORKSPACE_GOOGLE_PROJECT, WORKSPACE_NAME)
```

Comment:
Think we'll ever need the paginated version? https://github.com/broadinstitute/fiss/blob/46742e1ab7cbdc1b18e962d009a4a59715ade894/firecloud/api.py#L450

Reply:
Possibly, but maybe not right now.

I'll add an issue.

Reply:
See https://github.com/DataBiosphere/terra-notebook-utils/issues/21